File: epers-front-end/src/app/models/common/PaginatedListRequestModel.ts
export interface PaginatedListRequestModel {
    currentPage: number;
    itemsPerPage: number;
    idRol: number;
    idFirmaLoggedInUser: number | null;
    filter: string | null;
    filterFirma: number | null;
    matricolaLoggedInUser: string | null;
}

File: EpersBackend/Epers.Models/PaginatedListRequestModel.cs
namespace Epers.Models
{
    public class PaginatedListRequestModel
    {
        public int CurrentPage {get;set; }
        public int ItemsPerPage { get; set; } 
        public int IdRol { get; set; }
        public int? IdFirmaLoggedInUser { get; set; }
        public string? Filter { get; set; } 
        public int? FilterFirma { get; set; }  
        public string? MatricolaLoggedInUser { get; set; }
    }
}